plugins/azure/fix_plugin_azure/resource/cosmosdb.py

```diff
@@ -2000,6 +2010,7 @@ class AzureCosmosDBLocation(MicrosoftResource):
         "id": S("id"),
         "tags": S("tags", default={}),
         "name": S("name"),
+        "location": S("name"),
```

Comment:
why? I think this can be removed or?

Reply:
`AzureCosmosDBLocation` has these properties: 
```
  id: /subscriptions/subID/providers/Microsoft.DocumentDB/locations/westus2
  tags: {}
  name: West US 2
  ```
  so we can take location from the `name` property